src/AuthorizedSocketConnection.js

```diff
@@ -180,14 +180,13 @@ export default class AuthorizedSocketConnection<TContext, TCredentials> {
         document,
         null,
         {
-          subscribe: async (topic, { hasPermission, ...options } = {}) => {
+          subscribe: async (
+            topic,
+            { hasPermission = this.config.hasPermission, ...options } = {},
```

Comment:
this seems wrong for when there is no `options` passed in... `hasPermission` should still be `this.config.hasPermission` in that case

Reply:
that is what will happen, i think?

Reply:
hmm ok...